source/transfer_list.rst

```diff
@@ -657,5 +658,65 @@ such that the last ACPI table in this entry ends at offset
      - hdr_size
      - One or more ACPI tables.
 
+
+.. _tpm_evlog_entry:
+
+TPM info table entry layout (XFERLIST_TPM_INFO)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This entry type holds TPM-related information for a platform. The TPM event log
+info is a region containing a TPM event log as defined by TCG EFI Protocol
+Specification (https://trustedcomputinggroup.org/resource/tcg-efi-protocol-specification/).
+The CRB info defines the address of a region of memory that has been carved out
+and reserved for use as a TPM Command Response Buffer interface.
+
+.. _tab_tpm_info:
+.. list-table:: TPM info type layout
+   :widths: 2 2 2 8
+
+   * - Field
+     - Size (bytes)
+     - Offset (bytes)
+     - Description
+
+   * - tag_id
+     - 0x3
+     - 0x0
+     - The tag_id field must be set to **5**.
+
+   * - hdr_size
+     - 0x1
+     - 0x3
+     - |hdr_size_desc|
+
+   * - data_size
+     - 0x4
+     - 0x4
+     - 0x19
+
+   * - evlog_base_address
+     - 0x8
+     - 0x8
+     - The physical address of the region that contains the TPM event log.
+
+   * - crb_base_address
+     - 0x8
+     - 0x10
+     - The base address of CRB.
+
+   * - evlog_size 
+     - 0x4
+     - 0x18
+     - Size of the event log
+
+   * - crb_size 
+     - 0x4
+     - 0x1C
+     - Size of CRB. crb_size == 0 means that crb_base_address is not valid
+
+   * - need_to_replay
+     - 0x1
+     - 0x20
+     - Value "1" means bootloader needs to replay the EventLog. Default value is "0".    
```

Comment:
nit: Does the TPM specification define what exactly is meant by "replay"? I assume this means that the events stored in the log need to be written out to the TPM? This explanation could maybe use a little more detail, unless that is clear from context for everyone familiar with this specification.

Also, you might want to explicitly remind people here that the bootloader component that is doing this replay should update the TE to switch the flag from 1 to 0, so that a later bootloader component won't think it needs to be replayed again.

Reply:
I think this can be updated in the Description part.